app/templates/agenda/agendaitems/agendaitem/documents.hbs

```diff
@@ -44,6 +44,7 @@
       @onAddPiece={{perform this.addPiece}}
       @didDeleteContainer={{this.refresh}}
       @isEditable={{true}}
+      @decisionActivity={{this.decisionActivity}}
```

Comment:
Should we also add this to `app/templates/cases/case/subcases/subcase/documents.hbs`?

Reply:
I'm guessing we don't need to, I assume most of the document editing will happen from the agendaitem view & it's trivial to navigate there from the subcase view.

Reply:
I wouldn't, there can be more than 1 `decisionActivity` in this view if a subcase was postponed/reproposed.
Unless we can always assume this means the latest one...
Smells risky

Reply:
Ok, perfect, I used that same assumption in my own PR. We should find a way to encode/share these kinds of  assumptions, but that's a larger issue in our codebase than just here. Thanks for the info